tap-gui/plugins/runtime-resource-visibility.md

```diff
@@ -66,15 +65,82 @@ applicable for the resource type.
+
+Following, you can see an example of an expanded view for the index table.
+
+![Runtime resources index table collapsed](images/runtime-resources-expanded.png)
+
+## Detail pages
+
+Currently, we provide pages for many resources to allows users to see the most relevant characteristics of each one, including direct links to other resources.
+These are the common sections included in all pages:
+
+### Overview section
+
+It is the first card in all details pages, most of the information included in it came from the `metadata` attribute in each object. Some attributes displayed here are:
+
+1. **.YAML** button: Show the current object's definition in yaml; you can copy the full content using the icon in the top-right corner.
+2. Name
+3. Namespace
+4. Age / Creation date
+5. Cluster*
+6. URL**
+
+**Cluster: the value displayed corresponds to the name used in the cluster's configuration.*
+
+** _URL: Available for knative services and K8s services_
+
+E.g:
+
+![Overview section](images/runtime-resources-overview.png)
+
+### Status section
+
+This section displays the all the conditions included in the resource's attribute `status.conditions`; not all resources has conditions, they could be different between each one.
+
+For a better understanding about status please visit [Concepts - Object Spec and Status](https://kubernetes.io/docs/concepts/_print/#object-spec-and-status).
+
+![Status section](images/runtime-resources-status.png)
+
+### Ownership section
+
+Depending on the resource that you are currently viewing, this section presents all the resources specified in the `metadata.ownerReferences`.
+
+> For additional information about owners and dependants please visit [the official documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/)
+
+![Ownership section](images/runtime-resources-ownership.png)
+
+### Annotations & Labels
+
+Shows information on `metadata.annotations` and`metadata.labels`
+
+![Annotations and labels section](images/runtime-resources-annotations.png)
+
+## Navigating to pods
+
+You can go to pod's details page looking for the desired pod in the index table available in the first view by opening each of the resources there, like this:
+
+![Accessing pod from home](images/runtime-resources-index-pod.png)
+
+Or, you can use the table listing Pods in each detail page for components with a higher level in the Ownership hierarchy*, e.g:
+
+![Accessing pod from home](images/runtime-resources-pods.png)
+
+**Columns could be different on each detail page.*
+
+## Knative Service Details page
 
 To view the Knative services details of your resources, select the resource with 'Knative Service' type.
-In this page, additional information is available for Knative resources including status, an ownership hierarchy,
+In this page, additional information is available for Knative resources including status, an ownership hierarchy, 
 incoming routes, revisions, and pod details.
```

Comment:
Add a screenshot at the end to display the Application Live View integration with the pod page.
Application Live View integration helps developers to gain insights into the running pods they selected.